validator/src/main/java/com/amazon/aoc/validators/ValidatorFactory.java

```diff
@@ -45,8 +45,13 @@ public IValidator launchValidator(ValidationConfig validationConfig) throws Exce
         validator = new TraceValidator();
         expectedData = validationConfig.getExpectedTraceTemplate();
         break;
-      case "metric":
-        validator = new MetricValidator();
+      case "cw-metric":
+        validator = new CWMetricValidator();
+        expectedData = validationConfig.getExpectedMetricTemplate();
+        break;
+      case "prom-metric-scraping":
+      case "prom-metric-sd":
```

Comment:
what's the difference between "prom-metric-scraping" and "prom-metric-sd" from validator perspective?

Reply:
There is no difference between the `prom-metric-scraping` and `prom-metric-sd` testcases when instantiating the validators. It only instantiates different comparators later on in the actual validation.